code/game/objects/items/weapons/material/material_armor.dm

```diff
@@ -130,4 +130,24 @@ Protectiveness | Armor %
 	contained_sprite = 1
 
 /obj/item/clothing/head/helmet/material/makeshift/plasteel
-	default_material = "plasteel"
\ No newline at end of file
+	default_material = "plasteel"
+
+/obj/item/clothing/suit/armor/material/makeshift/kelly
```

Comment:
Rename the path to `makeshift/trenchcoat`. Searching for this in the spawn list will be a pita otherwise.